### PR TITLE
metadata improvements

### PR DESCRIPTION
**Pull request by tchughesiv**
/assign @ibek

Signed-off-by: tchughesiv <tchughesiv@gmail.com>

### CODE REVIEW

File: deploy/catalog_resources/community/1.0.1/kiecloud-operator.1.0.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -106,8 +106,8 @@ spec:
             path: deployments
             x-descriptors:
               - "urn:alm:descriptor:com.tectonic.ui:podStatuses"
-          - description: The address for accessing workbench, if it is deployed.
-            displayName: Workbench URL
+          - description: The address for accessing Business Central, if it is deployed.
+            displayName: Business Central URL
```
Comment (bmozaffa) [inline deploy/catalog_resources/community/1.0.1/kiecloud-operator.1.0.1.clusterserviceversion.yaml]: Business/Decision Central

File: deploy/catalog_resources/community/1.1.0/kiecloud-operator.1.1.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -106,8 +106,8 @@ spec:
             path: deployments
             x-descriptors:
               - "urn:alm:descriptor:com.tectonic.ui:podStatuses"
-          - description: The address for accessing workbench, if it is deployed.
-            displayName: Workbench URL
+          - description: The address for accessing Business Central, if it is deployed.
+            displayName: Business Central URL
```
Comment (bmozaffa) [inline deploy/catalog_resources/community/1.1.0/kiecloud-operator.1.1.0.clusterserviceversion.yaml]: Ditto

File: deploy/catalog_resources/community/1.0.1/kiecloud-operator.1.0.1.clusterserviceversion.yaml (hunk 1 above)
Comment (tchughesiv) [reply deploy/catalog_resources/community/1.0.1/kiecloud-operator.1.0.1.clusterserviceversion.yaml]: done

File: deploy/catalog_resources/community/1.1.0/kiecloud-operator.1.1.0.clusterserviceversion.yaml (hunk 2 above)
Comment (tchughesiv) [reply deploy/catalog_resources/community/1.1.0/kiecloud-operator.1.1.0.clusterserviceversion.yaml]: done

**Comment by tchughesiv** [comment]
@bmozaffa done

**Pull request merged**